src/TestSuite/TestCase.php

```diff
@@ -44,6 +44,13 @@ abstract class TestCase extends BaseTestCase
      */
     public $fixtureManager;
 
+    /**
+     * Fixtures used by this test case.
+     *
+     * @var array|string
+     */
+    public $fixtures = null;
```

Comment:
The = null is not necessary.
Also if null is still there at the end of constructor, |null should be added to docblock.